velox/duckdb/conversion/tests/DuckParserTest.cpp

```diff
@@ -129,6 +129,12 @@ TEST(DuckParserTest, in) {
   EXPECT_EQ(
       "in(\"col1\",[\"a\",null,\"b\",\"c\"])",
       parseExpr("col1 in ('a', null, 'b', 'c')")->toString());
+
+  ParseOptions options;
+  options.functionPrefix = "prefix";
```

Comment:
"prefixin" is hard to read. Perhaps, change to "prefix.in" or "prefix_in".

Let's make sure to test all functions which applies to.

Reply:
Fixed. Tests for other functions is WIP.